dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -2572,24 +2840,371 @@
       "x-ms-discriminator-value": "NumericRangeResolution",
       "properties": {
         "rangeKind": {
+          "$ref": "#/definitions/NumericRangeKind"
+        },
+        "minimum": {
+          "type": "number",
+          "format": "double",
+          "description": "The beginning value of  the interval."
+        },
+        "maximum": {
+          "type": "number",
+          "format": "double",
+          "description": "The ending value of the interval."
+        }
+      },
+      "required": [
+        "rangeKind",
+        "minimum",
+        "maximum"
+      ]
+    },
+    "BaseMd": {
+      "description": "The abstract base class for entity Metadata.",
+      "type": "object",
+      "discriminator": "metadataKind",
+      "properties": {
+        "metadataKind": {
+          "description": "The entity Metadata object kind.",
           "type": "string",
           "enum": [
-            "Number",
-            "Speed",
-            "Weight",
-            "Length",
-            "Volume",
-            "Area",
-            "Age",
-            "Information",
-            "Temperature",
-            "Currency"
+            "TemporalMd",
+            "NumberMd",
+            "OrdinalMd",
+            "SpeedMd",
+            "WeightMd",
+            "LengthMd",
+            "VolumeMd",
+            "AreaMd",
+            "AgeMd",
+            "InformationMd",
+            "TemperatureMd",
+            "CurrencyMd",
+            "NumberRangeMd",
+            "TemporalSpanMd"
           ],
           "x-ms-enum": {
-            "name": "RangeKind",
+            "name": "MetadataKind",
             "modelAsString": true
-          },
-          "description": "The kind of range that the resolution object represents."
+          }
+        }
+      },
+      "required": [
+        "metadataKind"
+      ]
+    },
+    "QuantityMd": {
```

Comment:
Same here: please avoid abbreviations or at least use an `x-ms-client-name` to direct the client libraries. I'll not repeat for brevity, so please check for other instances of just "Md".

Reply:
It looks like this has been addressed, so I will mark it as resolved.